packages/fxa-content-server/app/tests/spec/views/pair/auth_allow.js

```diff
@@ -45,8 +50,8 @@ describe('views/pair/auth_allow', () => {
     });
     user = new User();
     account = user.initAccount({ email: MOCK_EMAIL, uid: 'uid' });
-    sinon.stub(account, 'checkTotpTokenExists').callsFake(() => {
```

Comment:
As a side note, I think the `checkTotpTokenExists` helper function is no longer useful. The `/account/profile` returns the same data and more. I'll file a follow up issue.

Reply:
👍